app/controllers/provider_interface/offers_controller.rb

```diff
@@ -128,7 +128,7 @@ def update_conditions_service
         application_choice: @application_choice,
         standard_conditions: @wizard.standard_conditions,
         further_condition_attrs: @wizard.further_condition_attrs,
-        structured_conditions: @wizard.structured_conditions || @application_choice.offer.ske_conditions,
+        structured_conditions: @wizard.structured_conditions || @application_choice.offer.ske_conditions || [],
```

Comment:
wouldn't it be nice if `@application_choice.offer.ske_conditions` returned an empty array by default, so we don't have to do nil-checking here

Reply:
I think the issue is that the offer is nil (interviewing) for example. Maybe also add the & before ske_conditions?